WcaOnRails/app/views/delegate_reports/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for(@delegate_report) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :content %>
+  </div>
+
+  <div class="form-actions">
+    <%= f.button :submit %>
+  </div>
+<% end %>
```

Comment:
Nevermind! I see that's used in the `new.html.erb`
Anyway why wouldn't you change this a bit reuse in the `edit.html.erb`
If the above it's not the right way, I think we don't need this partial because it's used only once.